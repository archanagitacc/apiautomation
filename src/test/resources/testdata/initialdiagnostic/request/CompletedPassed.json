{"correlationId":"DynamicCorrelation","accessSeekerID":"ASI000000220777","msgType":"initialDiagnostic","data":{"avcId":"AVC600022276270", "problemSymptoms":"No Connection","ntd":{ "opticalStatus":"Red","opticalBehaviour":"Flashing","powerStatus":"Green","powerBehaviour":"Solid","internetStatus":"Red","internetBehaviour":"Flashing"}}}