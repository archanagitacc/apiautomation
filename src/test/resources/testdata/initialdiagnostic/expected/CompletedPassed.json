{
	"status": "COMPLETED",
	"testDetails": [{
			"testName": "ntdStatus",
			"status": "Completed",
			"result": "Passed"
		}
	]
}